tests/test_api.py

```diff
@@ -13,7 +13,7 @@ class FakturoidTestCase(unittest.TestCase):
 
     @patch('requests.post', return_value=response("token.json"))
     def setUp(self, mock):
-        self.fa = Fakturoid('myslug')
+        self.fa = Fakturoid('myslug', 'python-fakturoid-v3-tests (https://github.com/jarovo/python-fakturoid-v3)')
         self.fa.oauth_token_client_credentials_flow('pytest', b'client_id', b'client_secret')
```

Comment:
the oauth_token_client_credentials_flow should be changed to get the user agent from Fakturoid instance